ingest/ingest_pipeline.py

```diff
@@ -148,12 +156,14 @@ def __init__(
             self.cell_metadata_file = cell_metadata_file
 
     @my_debug_logger()
+    # Will be replaced by MongoConnection
     def get_mongo_db(self):
         host = os.environ['DATABASE_HOST']
         user = os.environ['MONGODB_USERNAME']
         password = os.environ['MONGODB_PASSWORD']
         db_name = os.environ['DATABASE_NAME']
-
+        print(f'db_name:{db_name},  user:{user}, host:{host}')
+        self.error_logger.error(f'db_name:{db_name},  user:{user}, host:{host}', extra=self.extra_log_params)
```

Comment:
Conventional spacing would enhance `grep`-ability.

```suggestion
        print(f'db_name: {db_name},  user: {user}, host: {host}')
        self.error_logger.error(f'db_name: {db_name},  user: {user}, host: {host}', extra=self.extra_log_params)
```